projects/cdk/schematics/ng-update/steps/migrate-sliders/migrate-input-slider.ts

```diff
@@ -0,0 +1,45 @@
+import {Tree} from '@angular-devkit/schematics';
+import {getComponentTemplates} from '../../../utils/templates/get-component-templates';
+import {DevkitFileSystem} from 'ng-morph/project/classes/devkit-file-system';
+import {addImportToNgModule} from 'ng-morph';
+import {addUniqueImport} from '../../../utils/add-unique-import';
+import {findNgModule} from '../../../utils/angular/ng-module';
+import {getNgComponents} from '../../../utils/angular/ng-component';
+import {replaceInputProperty} from '../../../utils/templates/ng-component-input-manipulations';
+
+export function migrateInputSlider(tree: Tree): void {
+    replaceSecondaryByTextfieldCustomContent(tree);
+}
+
+function replaceSecondaryByTextfieldCustomContent(tree: Tree): void {
```

Comment:
Probably we could make this function more general. It fits for `AbstractTuiInteractive` as well (many properties replaced by directives).

Reply:
I created new util `replaceInputPropertyByDirective`